Tests/Twig/ExtensionTest.php

```diff
@@ -37,33 +37,39 @@ public function testPiwikCodeDoesNotContainApiURL()
         $siteId = 1;
         $hostname = 'myHost.de';
         $path = 'piwik.php';
-        $extension = new Extension(false, $siteId, $hostname, $path);
+        $extension = new Extension(false, $siteId, $hostname, $path, true);
         $this->assertNotContains('_paq.push([\'setAPIUrl\', u]);', $extension->piwikCode());
     }
 
     public function testPiwikCodeContainsHostName()
     {
         $hostname = 'myHost.de';
-        $extension = new Extension(false, 1, $hostname, false);
+        $extension = new Extension(false, 1, $hostname, false, true);
         $this->assertContains($hostname, $extension->piwikCode());
     }
 
     public function testAdditionalApiCallsCanBeAdded()
     {
-        $extension = new Extension(false, 1, 'my.host', false);
+        $extension = new Extension(false, 1, 'my.host', false, true);
         $extension->piwikPush('foo', 'bar', 'baz');
         $this->assertContains('["foo","bar","baz"]', $extension->piwikCode());
     }
 
     public function testTrackPageViewEnabledByDefault()
     {
-        $extension = new Extension(false, 1, 'my.host', false);
+        $extension = new Extension(false, 1, 'my.host', false, true);
         $this->assertContains('"trackPageView"', $extension->piwikCode());
     }
 
+    public function testCookiesDisabledByDefault()
```

Comment:
When you make `false` the default parameter value (see suggestion below), this would probably rather be named `testCookiesCanBeDisabled`.